media/js/login.js

```diff
@@ -25,7 +25,13 @@
                     });
                     if ($form.data('refresh')) {
                         setTimeout(function() {
-                            window.location = './';
+                            if(window.location.hash != undefined){
```

Comment:
I don't think location.hash will ever be undefined (empty string in chrome). I could be wrong.